schwifty/bank_registry/manual_es.json

```diff
@@ -1,130 +1,1546 @@
+      "name": "BANCO ESPIRITO SANTO S A SUCUR",
+      "primary": true,
+      "short_name": "NOVO BANCO"
+  },
+  {
+      "bank_code": "0184",
+      "bic": "BEDFESM1XXX",
+      "country_code": "ES",
+      "name": "BANCO EUROPEO DE FINANZAS",
+      "primary": true,
+      "short_name": "BANCO EUROPEO DE FINANZAS"
+  },
+  {
+      "bank_code": "0220",
+      "bic": "FIOFESM1XXX",
+      "country_code": "ES",
+      "name": "BANCO FINANTIA SOFINLOC S.A.",
+      "primary": true,
+      "short_name": "BANCO FINANTIA"
+  },
+  {
+      "bank_code": "0232",
+      "bic": "INVLESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO INVERSIS NET S.A.",
+      "primary": true,
+      "short_name": "BANCO INVERSIS"
+  },
+  {
+      "bank_code": "0487",
+      "bic": "GBMNESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO MARE NOSTRUM",
+      "primary": true,
+      "short_name": "BANCO MARE NOSTRUM"
+  },
+  {
+      "bank_code": "0186",
+      "bic": "BFIVESBBXXX",
+      "country_code": "ES",
+      "name": "BANCO MEDIOLANUM, S.A.",
+      "primary": true,
+      "short_name": "BANCO MEDIOLANUM"
+  },
+  {
+      "bank_code": "0238",
+      "bic": "PSTRESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO PASTOR",
+      "primary": true,
+      "short_name": "BANCO PASTOR"
+  },
+  {
+      "bank_code": "0072",
+      "bic": "PSTRESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO PASTOR",
+      "primary": true,
+      "short_name": "BANCO PASTOR"
+  },
+  {
+      "bank_code": "0235",
+      "bic": "PICHESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO PICHINCHA ESPANA",
+      "primary": true,
+      "short_name": "BANCO PICHINCHA"
+  },
+  {
+      "bank_code": "0004",
+      "bic": "POPUESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO POPULAR",
+      "primary": true,
+      "short_name": "BANCO POPULAR"
+  },
+  {
+      "bank_code": "0024",
+      "bic": "POPUESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO POPULAR",
+      "primary": true,
+      "short_name": "BANCO POPULAR"
+  },
+  {
+      "bank_code": "0082",
+      "bic": "POPUESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO POPULAR",
+      "primary": true,
+      "short_name": "BANCO POPULAR"
+  },
+  {
+      "bank_code": "0095",
+      "bic": "POPUESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO POPULAR",
+      "primary": true,
+      "short_name": "BANCO POPULAR"
+  },
+  {
+      "bank_code": "0097",
+      "bic": "POPUESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO POPULAR",
+      "primary": true,
+      "short_name": "BANCO POPULAR"
+  },
+  {
+      "bank_code": "0116",
+      "bic": "POPUESMMXXX",
+      "country_code": "ES",
+      "name": "BANCO POPULAR",
+      "primary": true,
+      "short_name": "BANCO POPULAR"
+  },
+  {
+      "bank_code": "0081",
+      "bic": "BSABESBBXXX",
+      "country_code": "ES",
+      "name": "BANCO SABADELL",
```

Comment:
How about 
```json
{
  "name": "Banco de Sabadell, S.A.",
  "short_name": "Banco Sabadell"
}
```
This can be used as a general pattern as distinction between `"name"` and `"short_name"`. The `"name"` would then include the legal entity type (e.g. S.A.) and the `"short_name"` just the commonly used name.